swap_meet/electronics.py

```diff
@@ -1,2 +1,14 @@
-class Electronics:
-    pass
+from .item import Item
+
+class Electronics(Item):
+
+    def __init__(self, id=None, condition=0, age=0, type="Unknown"):
+        super().__init__(id, condition, age)
+        self.type = type
+
+    def get_category(self):
+        return "Electronics"
+
+    def __str__(self):
+        return f"An object of type {self.get_category()} with id {self.id}. This is a {self.type} device."
```

Comment:
👀 This exhibits the same potential for reusing the parent implementation as `Clothing`.